inst/app/www/custom.css

```diff
@@ -48,9 +48,8 @@
 }
 
 .title-panel img.UOB-logo {
-  height: 40px;
+  height: 80px;
   /* Slightly smaller size for the UOB logo */
-  margin-left: 10px;
   /* Add spacing between logos */
```

Comment:
Are the comments below the line it refers to, or do we have an extra comment here?